pkg/kp/kv.go

```diff
@@ -171,3 +222,16 @@ func (c consulStore) Ping() error {
 	}
 	return nil
 }
+
+func addTimeStamp(value WatchResult) (time.Time, WatchResult) {
+	currentTime := time.Now()
+	if value.Time == "" {
+		value.Time = currentTime.String()
+	}
+
+	return currentTime, value
+}
+
+func healthPath(service, node string) string {
+	return fmt.Sprintf("%s%s/%s", service, "health", node)
```

Comment:
its servicehealth/nodename. It could just be service/node or service/health/node